frontend/src/app/pages/DashBoardPage/utils/board.ts

```diff
@@ -203,3 +204,23 @@ export const getChartDataView = (views: View[], dataCharts: DataChart[]) => {
   });
   return viewViews;
 };
+
+/**
+ * 
+ * @date 2022-02-09
+ * @param {any} widgets:Widget[]
```

Comment:
VSCode Plugins `Document This`, should be help to quick create description with JSDoc annotation.

Reply:
good advice